### PR TITLE
Convert HTML to markdown on Vagrant configure page

### PR DESCRIPTION
**Pull request by leviv**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [x] Bug fix or improvement

## Summary
Converts remaining HTML tags to MD.

When this pull request is merged, it will...
Make devdocs source code easier to read.

Converts HTML to MD

### CODE REVIEW

File: guides/v2.1/config-guide/varnish/config-varnish.md [hunk 1]
```diff
@@ -52,21 +51,19 @@ The process can be summarized as follows:
 
 	If there is nothing in your `<your Magento install dir>/var/page_cache` directory, you've successfully configured Varnish with Magento!
 
-<div class="bs-callout bs-callout-info" id="info">
-	<ul><li>Except where noted, you must enter all commands discussed in this topic as a user with <code>root</code> privileges.</li>
-		<li>This topic is written for Varnish on CentOS and Apache 2.2. If you're setting up Varnish in a different environment, some commands are likely different. Consult Varnish documentation for more information.</li></ul>
-</div>
+{:.bs-callout .bs-callout-info #info}
```
Comment (shrielenee) [inline guides/v2.1/config-guide/varnish/config-varnish.md]: Make sure there is no blank line or hard return in this note, as it won't render properly.

File: guides/v2.2/config-guide/varnish/config-varnish.md [hunk 2]
```diff
@@ -13,51 +10,49 @@ functional_areas:
...
+{:.bs-callout .bs-callout-warning}
+We _strongly recommend_ you use Varnish in production. The built-in full-page caching (to either the file system or [database]) is much slower than Varnish, and Varnish is designed to accelerate HTTP traffic.
 
 For more information about Varnish, see:
 
-*	<a href="https://en.wikipedia.org/wiki/Varnish_%28software%29" target="_blank">wikipedia</a>
-*	<a href="https://www.varnish-cache.org/docs/trunk/users-guide/intro.html" target="_blank">The Big Varnish Picture</a>
-*	<a href="https://www.varnish-cache.org/docs/trunk/reference/varnishd.html#ref-varnishd-options" target="_blank">Varnish startup options</a>
-*	<a href="https://www.varnish-cache.org/docs/trunk/users-guide/performance.html#users-performance" target="_blank">Varnish and Website Performance</a>
+*	[wikipedia]
```
Comment (shrielenee) [inline guides/v2.2/config-guide/varnish/config-varnish.md]: Capitalize "Wikipedia" - in other file as well.

File: guides/v2.1/config-guide/varnish/config-varnish.md (hunk 1 above)
Comment (leviv) [reply guides/v2.1/config-guide/varnish/config-varnish.md]: The note seems to be rendering correctly for me

File: guides/v2.2/config-guide/varnish/config-varnish.md (hunk 2 above)
Comment (leviv) [reply guides/v2.2/config-guide/varnish/config-varnish.md]: Fixed

**Comment by leviv** [comment]
@shrielenee made changes

File: guides/v2.2/config-guide/varnish/config-varnish.md (hunk 2 above)
Comment (shrielenee) [inline guides/v2.2/config-guide/varnish/config-varnish.md]: Still not capitalized here or in the previous file.

Comment (shrielenee) [reply guides/v2.2/config-guide/varnish/config-varnish.md]: I will make the needed capitalization change.

**Comment by shrielenee** [comment]
Many thanks @leviv!

**Pull request merged**